src/ci/resources/gcp_batch_provider_config.inc.conf

```diff
@@ -3,6 +3,8 @@ root = "gs://cloud-cromwell-dev-self-cleaning/cromwell_execution/ci"
 maximum-polling-interval = 600
 concurrent-job-limit = 1000
 
+quota-attempts: 3
```

Comment:
Ah, nope, good call. This is a leftover from when I was spamming the key everywhere because I couldn't get it to read and thought maybe my instance was reading the wrong config.